index.html

```diff
@@ -29,10 +29,186 @@ <h1 class="content-text">I'm Andrea Glad to see you!</h1>
+                      </p>
+                      <ul class="badges">
+                        <li class="works-badge">html</li>
+                        <li class="works-badge">css</li>
+                        <li class="works-badge">javascript</li>
+                      </ul>
+                      <button class="btn" type="button">See Project</button>
+                    </div>
+                  </div>
+        
+                  <div class="work-card">
+                  <div>
+                      <img src="SnapshootPortfoliotonic01.png" alt="" />
+                    </div>
+                    <div class="align-center-text">
+                      <h3 class="work-heading">Tonic</h3>
+                      <p class="dev-date mt-2">
+                        <b class="canopy">CANOPY</b>
+                        <span class="bullet">&#9679;</span>Back End Dev
+                        <span class="bullet">&#9679;</span> 2015
+                      </p>
+                      <p class="work-description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                      </p>
+                      <ul class="badges">
+                        <li class="works-badge">html</li>
+                        <li class="works-badge">css</li>
+                        <li class="works-badge">javascript</li>
+                      </ul>
+                      <button class="btn" type="button">See Project</button>
+                    </div>
+                  </div>
+        
+                  <div class="work-card">
+                  <div>
+                      <img src="SnapshootPortfoliomps01.png" alt="" />
+                    </div>
+                    <div class="align-center-text">
+                      <h3 class="work-heading">Multi-Post Stories</h3>
+                      <p class="dev-date mt-2">
+                        <b class="canopy">CANOPY</b>
+                        <span class="bullet">&#9679;</span>Back End Dev
+                        <span class="bullet">&#9679;</span> 2015
+                      </p>
+                      <p class="works-content-p">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                      </p>
+                      <ul class="badges">
+                        <li class="works-badge">html</li>
+                        <li class="works-badge">css</li>
+                        <li class="works-badge">javascript</li>
+                      </ul>
+                      <button class="btn" type="button">See Project</button>
+                    </div>
+                  </div>
+                  </div>
+              </section>
+              <section class="about-section-2">
+                <div class="align-center-text">
+                  <h1 class="header-text">About<br>Myself</h1>
```

Comment:
- [ ] Please align the items in the about me section using flexbox
![image](https://user-images.githubusercontent.com/74744787/185765135-d07b42ba-076e-4120-be1d-bee6c5db6bdc.png)